server/util/ipfs.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import axios from 'axios';
+const FormData = require('form-data');
+import models from '../database';
+require('dotenv').config();
+
+const ipfs = async (userID, addressID, jsonfile) => {
+  const data = new FormData();
+  data.append('file', JSON.stringify(jsonfile), 'userIDblob');
+  const headers = {
+    pinata_api_key: process.env.PINATA_API_KEY,
+    'Content-Type': `multipart/form-data; boundary= ${data._boundary}`,
+    pinata_secret_api_key: process.env.PINATA_SECRET_API_KEY,
+  };
+
+  const response = await axios
+    .post('https://api.pinata.cloud/pinning/pinFileToIPFS', data, { headers })
+    .then(async (response) => {
```

Comment:
Please clean this up a little -- we shouldn't be mixing then/catch and async/await blocks. You can straightforwardly convert the former to the latter.

Reply:
Done